plugins/modules/create_blueprint.py

```diff
@@ -39,12 +39,12 @@
         type: str
         required: false
         default: ""
-    version:
+    version_type:
         description:
-            - Semantic Versioned (https://semver.org/) version number
+            - Semantic Versioned (https://semver.org/) version type
```

Comment:
It was to reference the type of versioning the user wants to update with. Is it a major , minor , or patch update.

I changed it to `Specify the semantic version. Ref (https://semver.org/)` . In the doc it references them as simply a version. `MAJOR version`. So I think this fits better? :man_shrugging:

Reply:
I would explain what `version_type` means because reading this sentence without any context I honestly don't know what it does mean or the intended purpose of the value "patch" does for me as an user. You can be as verbose as necessary here, it doesn't have to be a short explanation.